src/pages/Chat/ResetButton.tsx

```diff
@@ -1,8 +1,15 @@
+import { TranslationsConsumer } from '@hedviginsurance/textkeyfy'
 import * as React from 'react'
 import { ChatContainer } from './state'
 
 export const ResetButton = () => (
   <ChatContainer>
-    {({ reset }) => <button onClick={reset}>reset session</button>}
+    {({ reset }) => (
+      <button onClick={reset}>
+        <TranslationsConsumer textKey="CHAT_RESET_LABEL">
```

Comment:
matter of taste, but i tend to wrap the component needing text as a child, so doing:

```
<Consumer>
{text => <button>{text}</button>}
</Consumer>
```

But I'm fine with whatever :)

Reply:
haha i think i've done mix, nice catch. Lets be consistent